simulation/navigator_gazebo/models/wamv/wamv.sdf

```diff
@@ -183,7 +192,7 @@
 
       <sensor name="velodyne_lidar" type="ray">
         <visualize>false</visualize>
-        <pose>-0.277622 0 0.648208 0.0 0.0 0.0</pose>
+        <pose> -0.277622 0.0 0.648208 0.0 -0.0 0.0 </pose>
```

Comment:
I think having it inaccurate is fine for now. We'll add in a tf once we actually have the camera on the boat, and update the sdf using the tool from the tf.